### ISSUE TITLE
simpleTradingBan

### ISSUE THREAD

**Issue by PraseodymiumSpike** [fairy-stockfish/Fairy-Stockfish#690]
It would be nice to have something called simpleTradingBan that accepts a piece list and disallows those pieces to be traded with themselves using chu shogi lion anti-trading rules. For example,
```[nomajortrades:chess]
simpleTradingBan = qr```
would make it so that a queen couldn't be traded for a queen, and a rook couldn't be traded for a rook. But a rook could be traded for a queen. This option would not work in variants with captureToHand on.

**Comment by ianfab** [comment]
Isn't that the same as https://github.com/fairy-stockfish/Fairy-Stockfish/pull/663/files#diff-33c5d3b051dea86f192d5315580529b171d25cb80f51c5c9db0fb2c603129846R170 ?

**Comment by PraseodymiumSpike** [comment]
No, they are very different. The anti-trading rules are way more sophisticated than that. You can find more about them here: https://www.chessvariants.com/rules/chu-shogi

**Comment by ianfab** [comment]
Ok, thanks for the clarification. Considering that chu shogi is to my knowledge the only established variant having that and it anyway currently is impossible for other reasons, I do not really see a reason to add it though (not mentioning my subjective aversion to this rule).
